controllers/mover/syncthing/syncthing_test.go

```diff
@@ -474,7 +477,143 @@ var _ = Describe("When an RS specifies Syncthing", func() {
 		})
 
 		Context("Syncthing API is being used properly", func() {
-			// TODO: test syncthing API
+			When("Syncthing server does not exist", func() {
+				It("Doesn't synchronize", func() {
+					res, err := mover.Synchronize(ctx)
+					Expect(err).ToNot(BeNil())
```

Comment:
It's been awhile since I've used gomega but is the nil check necessary? IOW, do you need to check for nil before you can check for expected errors?

Reply:
You don't need to necessarily do one before the other, but having them helps assert the behavior of the code itself and what we expect it to do under the given conditions.